ci-operator/config/openshift/lightspeed-service/OWNERS

```diff
@@ -0,0 +1,15 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift/library-go root OWNERS
```

Comment:
this doesn't seem right....

Reply:
Yes you are right. I just forgot to remove those comments from the template code. It wasn't auto-generated. I created these files manually.